cmd/skupper/skupper_podman_site.go

```diff
@@ -1,20 +1,131 @@
 package main
 
 import (
+	"fmt"
+
 	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/client/container"
+	"github.com/skupperproject/skupper/pkg/domain"
+	podman "github.com/skupperproject/skupper/pkg/domain/podman"
+	"github.com/skupperproject/skupper/pkg/qdr"
 	"github.com/spf13/cobra"
 )
 
-type SkupperPodmanSite struct{}
+type SkupperPodmanSite struct {
+	podman *SkupperPodman
+	flags  PodmanInitFlags
+}
+
+type PodmanInitFlags struct {
+	IngressHosts               []string
+	IngressBindIPs             []string
+	IngressBindInterRouterPort int
+	IngressBindEdgePort        int
+	ContainerNetwork           string
+	PodmanEndpoint             string
+}
 
 func (s *SkupperPodmanSite) Create(cmd *cobra.Command, args []string) error {
-	return notImplementedErr
+	siteName := routerCreateOpts.SkupperName
+	if siteName == "" {
+		siteName = podman.Username
+	}
+	// fmt.Printf("site name         : %s\n", siteName)
```

Comment:
Is this block something temporary?

Reply:
I have to clean up those debug messages.

Reply:
... or are you referring to the siteName being the username?

Reply:
No, I meant the cleaning the block of debug messages like you said earlier (sorry for the delay).